README.md

```diff
@@ -1,591 +1,151 @@
-```
-
-The `$install` operation is triggered with a POST to `[server]/ImplementationGuide/$install`, with the payload below:
 
+5. It should return something like:
 ```json
 {
-  "resourceType": "Parameters",
-  "parameter": [
-    {
-      "name": "npmContent",
-      "valueBase64Binary": "[BASE64_ENCODED_NPM_PACKAGE_DATA]"
-    }
-  ]
+   "resourceType": "Patient",
+   "id": "3",
+   "meta": {
+      "versionId": "1",
+      "lastUpdated": "2024-09-06T09:41:59.937+02:00",
+      "source": "#GdSt18FL7qntkzYu"
+   },
+   "identifier": [ {
+      "system": "http://hospital.org/patients",
+      "value": "12345"
+   } ],
+   "name": [ {
+      "use": "official",
+      "family": "Doe",
+      "given": [ "John" ]
+   } ],
+   "gender": "male",
+   "birthDate": "1980-01-01"
 }
 ```
 
-## Enable OpenTelemetry auto-instrumentation
-
-The container image includes the [OpenTelemetry Java auto-instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation)
-Java agent JAR which can be used to export telemetry data for the HAPI FHIR JPA Server. You can enable it by specifying the `-javaagent` flag,
-for example by overriding the `JAVA_TOOL_OPTIONS` environment variable:
-
-```sh
-docker run --rm -it -p 8080:8080 \
-  -e JAVA_TOOL_OPTIONS="-javaagent:/app/opentelemetry-javaagent.jar" \
-  -e OTEL_TRACES_EXPORTER="jaeger" \
-  -e OTEL_SERVICE_NAME="hapi-fhir-server" \
-  -e OTEL_EXPORTER_JAEGER_ENDPOINT="http://jaeger:14250" \
-  docker.io/hapiproject/hapi:latest
-```
-
-You can configure the agent using environment variables or Java system properties, see <https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/> for details.
+Before you request the timeline service ensure your endpoint is registered in the [address](https://github.com/minvws/nl-irealisatie-zmodules-addressing-register) and
+the [pseudonym-exchange-service](https://github.com/minvws/nl-irealisatie-zmodules-pseudonym-service) knows your provider-id.
+To find your provider-id, look in the [settings file](src/main/resources/application.yaml).
+
+In the address DB you should have a entry that sets the address endpoint to: http://host.docker.internal:8080/fhir
+And to set up the provider in the pseudonym service follow these steps:
+1. do the following curl request in the terminal
+```curl
+curl -X 'POST' \
+  'http://localhost:8504/register' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "provider_id": "MY_HAPI",
+  "bsn_hash": "2b826afa1c71f571a007dad61ce9fb9b63a24a328aa4ac009484a198ff565c51"
+}'
+```
+2. Copy generated pseudonym
+3. Open database editor to the `hapi` database
+4. Go to table `hfj_res_ver`
+5. Edit the extension in the `patient` resource entry
+6. Replace the patient's extensions UUID with the copied pseudonym
+
+Finally test whether this is all setup correctly:
+1. Do curl request:
+```curl
+curl 'http://localhost:8080/fhir/ImagingStudy/_search?pseudonym=677b33c7-30e0-4fe1-a740-87fd73c4dfaf'
```

Comment:
maybe we should mention that the following psuedonym is already setup as an example in the other service